java/configuration/src/main/java/sleeper/configuration/Requirements.java

```diff
@@ -34,11 +34,10 @@ private Requirements() {
      * the CPU requirement in the left element and memory requirement in the right element.
      *
      * @param  architecture       CPU architecture
-     * @param  launchType         the container launch type
      * @param  instanceProperties Sleeper instance properties
      * @return                    CPU and memory requirements as per the CPU architecture
      */
-    public static Pair<Integer, Integer> getArchRequirements(String architecture, String launchType, InstanceProperties instanceProperties) {
+    public static Pair<Integer, Integer> getArchRequirements(String architecture, InstanceProperties instanceProperties) {
```

Comment:
This is causing compilation failures in the CDK module.

Reply:
Thanks, fixed